src/dune_rules/coq/coq_lib.ml

```diff
@@ -154,6 +149,8 @@ module DB = struct
            Coq_lib_name.t
         -> [ `Redirect of t
            | `Theory of Lib.DB.t * Path.Build.t * Coq_stanza.Theory.t
+           | `Stdlib of lib
```

Comment:
let's make this `User_contrib legacy_lib`, for sure we are gonna save time in rules

Reply:
WDYM?

Reply:
I mean to have a type 
```
  type legacy_lib =
    { boot : t option Resolve.t
    ; id : Id.t
    ; implicit : bool (* Only useful for the stdlib *)
    ; installed_root : Path.Build.t
    }
```
plus the corresponding variants if we need them.

Reply:
So then for example how to properly generate flags like for plugins becomes clear.

Reply:
I see, I can try to see what I can do.